src-docs/src/views/tour/tour_example.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+
+import { renderToHtml } from '../../services';
+
+import { GuideSectionTypes } from '../../components';
+
+import {
+  EuiCode,
+  EuiSpacer,
+  EuiText,
+  EuiTourStep,
+} from '../../../../src/components';
+
+import Step from './step';
+import Tour from './tour';
+import Managed from './managed';
+import ManagedHook from './managed_hook';
+import FullScreen from './fullscreen';
+
+const stepSource = require('!!raw-loader!./step');
+const stepHtml = renderToHtml(Step);
+const stepSnippet = `
+<EuiTourStep
+  content={
+    <EuiText>
+      <p>The tour step content.</p>
+    </EuiText>
+  }
+  isStepOpen={true}
+  isTourActive={true}
+  minWidth={300}
+  onSkip={() => alert('Skip the tour!')}
+  onEnd={() => alert('End the tour!')}
+  step={1}
+  stepsTotal={1}
+  title="An individual step"
+  subtitle="A tour"
+  anchorPosition="rightUp">
+  <EuiText>
+    <p>The tour step anchor point.</p>
+  </EuiText>
+</EuiTourStep>
+
+`;
+const tourSource = require('!!raw-loader!./tour');
+const tourHtml = renderToHtml(Tour);
+const managedSource = require('!!raw-loader!./managed');
+const managedHtml = renderToHtml(Managed);
+const managedHookSource = require('!!raw-loader!./managed_hook');
+const managedHookHtml = renderToHtml(ManagedHook);
+
+const fullSource = require('!!raw-loader!./fullscreen');
+const fullHtml = renderToHtml(FullScreen);
+
+export const TourExample = {
+  title: 'Tour',
+  beta: true,
+  intro: (
+    <EuiText>
+      <p>
+        The tour components provided by EUI allow for a flexible and
+        customizable way to showcase items on a page in an ordered manner by
+        augmenting existing elements on the page without altering functionality.
+      </p>
+      <p>
+        The examples on this page, use <EuiCode>localStorage</EuiCode> to
+        persist state to demonstrate starting a tour at differnt stages.
+      </p>
+      <EuiSpacer />
+    </EuiText>
+  ),
+  sections: [
+    {
+      title: 'Step options',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: stepSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: stepHtml,
+        },
+      ],
+      text: (
+        <>
+          <p>
+            The <strong>EuiTourStep</strong> component is the base for building
+            a feature tour or individual popover for onboarding.
+          </p>
+          <p>
+            All content and actions—including titles, headings, and buttons—are
+            customizable via props.
```

Comment:
Is this statement true? I don't see a way to change the "End tour" or remove the dots. Unless it has to do with the `footerAction` prop which doesn't have any description.
```suggestion
            All content and actions including titles, headings, and buttons are
            customizable via props.
```